app/Http/Controllers/UserController.php

```diff
@@ -67,4 +69,16 @@ public function updateUser(updateUserRequest $request, User $user)
     {
         $this->userService->updateUser($request->all(), $user);
     }
+
+    public function getUserEmails()
+    {
+        $emails = $this->userService->getUserEmails();
+        return response()->json(['result' => true, 'emails' => $emails]);
```

Comment:
we shouldn't return 'result' or something like this. Users will get emails and status 200 which is enough.
`return response()->json(['emails' => $emails]);`